cypress-browser-agent-otel/calculator-app/assets/index.d1ad8cf4.js

```diff
@@ -1,2 +1,3 @@
otocols:["http","https","file","blob","url","data"]};function Wc(e,t){return gn(e,new Le.classes.URLSearchParams,Object.assign({visitor:function(n,r,s,i){return Le.isNode&&p.isBuffer(n)?(this.append(r,n.toString("base64")),!1):i.defaultVisitor.apply(this,arguments)}},t))}function Jc(e){return p.matchAll(/\w+|\[(\w*)]/g,e).map(t=>t[0]==="[]"?"":t[1]||t[0])}function Vc(e){const t={},n=Object.keys(e);let r;const s=n.length;let i;for(r=0;r<s;r++)i=n[r],t[i]=e[i];return t}function Ci(e){function t(n,r,s,i){let o=n[i++];const l=Number.isFinite(+o),u=i>=n.length;return o=!o&&p.isArray(s)?s.length:o,u?(p.hasOwnProp(s,o)?s[o]=[s[o],r]:s[o]=r,!l):((!s[o]||!p.isObject(s[o]))&&(s[o]=[]),t(n,r,s[o],i)&&p.isArray(s[o])&&(s[o]=Vc(s[o])),!l)}if(p.isFormData(e)&&p.isFunction(e.entries)){const n={};return p.forEachEntry(e,(r,s)=>{t(Jc(r),s,n,0)}),n}return null}function Yc(e,t,n){const r=n.config.validateStatus;!n.status||!r||r(n.status)?e(n):t(new U("Request failed with status code "+n.status,[U.ERR_BAD_REQUEST,U.ERR_BAD_RESPONSE][Math.floor(n.status/100)-4],n.config,n.request,n))}const Xc=Le.isStandardBrowserEnv?function(){return{write:function(n,r,s,i,o,l){const u=[];u.push(n+"="+encodeURIComponent(r)),p.isNumber(s)&&u.push("expires="+new Date(s).toGMTString()),p.isString(i)&&u.push("path="+i),p.isString(o)&&u.push("domain="+o),l===!0&&u.push("secure"),document.cookie=u.join("; ")},read:function(n){const r=document.cookie.match(new RegExp("(^|;\\s*)("+n+")=([^;]*)"));return r?decodeURIComponent(r[3]):null},remove:function(n){this.write(n,"",Date.now()-864e5)}}}():function(){return{write:function(){},read:function(){return null},remove:function(){}}}();function Qc(e){return/^([a-z][a-z\d+\-.]*:)?\/\//i.test(e)}function Zc(e,t){return t?e.replace(/\/+$/,"")+"/"+t.replace(/^\/+/,""):e}function Ti(e,t){return e&&!Qc(t)?Zc(e,t):t}const Gc=Le.isStandardBrowserEnv?function(){const t=/(msie|trident)/i.test(navigator.userAgent),n=document.createElement("a");let r;function s(i){let o=i;return t&&(n.setAttribute("href",o),o=n.href),n.setAttribute("href",o),{href:n.href,protocol:n.protocol?n.protocol.replace(/:$/,""):"",host:n.host,search:n.search?n.search.replace(/^\?/,""):"",hash:n.hash?n.hash.replace(/^#/,""):"",hostname:n.hostname,port:n.port,pathname:n.pathname.charAt(0)==="/"?n.pathname:"/"+n.pathname}}return r=s(window.location.href),function(o){const l=p.isString(o)?s(o):o;return l.protocol===r.protocol&&l.host===r.host}}():function(){return function(){return!0}}();function Mt(e,t,n){U.call(this,e==null?"canceled":e,U.ERR_CANCELED,t,n),this.name="CanceledError"}p.inherits(Mt,U,{__CANCEL__:!0});function eu(e){const t=/^([-+\w]{1,25})(:?\/\/|:)/.exec(e);return t&&t[1]||""}const tu=p.toObjectSet(["age","authorization","content-length","content-type","etag","expires","from","host","if-modified-since","if-unmodified-since","last-modified","location","max-forwards","proxy-authorization","referer","retry-after","user-agent"]),nu=e=>{const t={};let n,r,s;return e&&e.split(`
```

Comment:
I think these should be removed from the PR

Reply:
The previous assets for Cypress/Gauge were missing the sourcemaps so I updated them in this PR as well that's why those files are updated too